composer.json

```diff
@@ -23,18 +23,21 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5",
         "zendframework/zend-modulemanager": "2.*",
-        "zendframework/zend-stdlib": "2.*",
-        "zendframework/zend-servicemanager": "2.*",
+        "zendframework/zend-stdlib": "2.* || 3.*",
+        "zendframework/zend-servicemanager": "^2.7.5 || 3.*",
         "zendframework/zend-loader": "2.*",
-        "zendframework/zend-eventmanager": "2.*",
-        "zendframework/zend-mvc": "2.*",
+        "zendframework/zend-eventmanager": "2.* || 3.*",
+        "zendframework/zend-mvc": "2.* || 3.*",
         "kriswallsmith/assetic": "~1.2.1"
     },
     "require-dev": {
-        "zendframework/zendframework": "2.*",
-        "phpunit/phpunit": "~4.0",
+        "zendframework/zend-http": "2.*",
+        "zendframework/zend-view": "2.*",
+        "zendframework/zend-cache": "2.*",
+        "zendframework/zend-console": "2.*",
+        "phpunit/phpunit": "~4.8",
         "squizlabs/php_codesniffer": "1.5.*"
     },
```

Comment:
Also, I'd add a suggestion for zendframework/zend-mvc-console, if the user is using zend-mvc v3; the new package contains the mvc/console integration point, which are no longer present in zend-mvc v3.